.github/workflows/build-world.yaml

```diff
@@ -0,0 +1,96 @@
+name: Build the world
+
+on:
+  workflow_dispatch:
+    
+jobs:
+  build:
+    name: Build OS packages
+    runs-on:
+      - self-hosted
+      - ${{ matrix.arch }}
+
+    # Only run one build per arch at a time
+    concurrency:
+      group: build-world-${{ matrix.arch }}
+
+    strategy:
+      matrix:
+        arch: [ "x86_64", "arm64" ]
```

Comment:
Maybe the ship has sailed on this, but aarch64 might be more consistent with APK's sense of architectures.

Reply:
I can/will change it!